.github/ISSUE_TEMPLATE/offboarding.md

```diff
@@ -9,13 +9,14 @@ assignees: ''
 # Offboarding
 
 By creating this ticket, you are requesting a employee/contractor to be
```

Comment:
```suggestion
By creating this ticket, you are requesting an employee to be
```

We struck the contractor term from the onboarding template. So we need to do the same here.